tern/formats/cyclonedx/cyclonedxjson/package_helpers.py

```diff
@@ -15,7 +15,7 @@ def get_package_dict(os_guess, package):
     ''' Given a package format, namespace and package object return a
     CycloneDX JSON dictionary representation of the package. '''
     package_dict = {
-        'name': package.name,
```

Comment:
can we just return `package.name.lower()` here? Are there any purls that require uppercase?

Reply:
Among the packages manager supported by Tern, gem and pacman do not have specifications on their name in the purl-spec.
Also if you want to extend to other packages managers, some of them such as composer or maven do not specification on the name. And some of them should have uppercase such as nugget.